src/web/routes/application/postcode/postcode.js

```diff
@@ -1,3 +1,29 @@
+const request = require('request-promise')
+const { wrapError } = require('../common/formatters')
+const { transformOsPlacesApiResponse } = require('./adapters')
+
+const OS_PLACES_API_URL = 'https://api.ordnancesurvey.co.uk/places/v1/addresses/postcode'
```

Comment:
Might we need to make this configurable, potentially add it as an environment variable?

Reply:
If you add `OS_PLACES_URI: getVCAPServiceOSPlacesVariable('OS_PLACES_URI', process.env.OS_PLACES_URI)` to environments.js that should get it.

Reply:
Added this to config